### PR TITLE
Review of unreviewed guides

### PR DESCRIPTION
**Pull request by jstaffor**
New workflow:
[jstaffor@jstaffor raincatcher-docs]$ git reset --hard origin/master 
HEAD is now at 54cf080 Review7 guides (#107)

[jstaffor@jstaffor raincatcher-docs]$ git clean -f

[jstaffor@jstaffor raincatcher-docs]$ git status
On branch master
nothing to commit, working directory clean

[jstaffor@jstaffor raincatcher-docs]$ clear

[jstaffor@jstaffor raincatcher-docs]$ git co -b UnreviewedFiles
Switched to a new branch 'UnreviewedFiles'

[jstaffor@jstaffor raincatcher-docs]$ git status
On branch UnreviewedFiles
nothing to commit, working directory clean

### CODE REVIEW

**Review by jstaffor — COMMENTED** [review]
@wtrocki  - ready for review

File: docs/workforce-management-framework/topics/pro-contribute-guide.adoc [hunk 1]
```diff
@@ -80,4 +80,4 @@ Open a new terminal and navigate to the folder containing _raincatcher-angularjs
 npm run start
 ----
 
-Both the _Mobile_ and _Portal_ demo applications open automatically in a browser.
+Both the Mobile and Portal demo applications open automatically in a browser.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-contribute-guide.adoc]: Let's drop `demo` - we probably missed that previously. It's not your change but best to remove it when we see it.

File: docs/workforce-management-framework/topics/ref-mobile.adoc [hunk 2]
```diff
@@ -1,53 +1,56 @@
...
 
-===== `src/app/app.js`
-Has a small AngularJS controller for displaying a template before data is loaded from the backend server.
+|`src/sass`
+|Contains app-wide [Sass](http://sass-lang.com/) files that are compiled by the build pipeline.
 
-===== `src/sass`
-Contains app-wide [Sass](http://sass-lang.com/) files that are compiled by the build pipeline.
+|`src/app/initialization`
+|Contains configuration values and initial bootstrapping code for the Angularjs application.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/ref-mobile.adoc]: AngularJS

**Review by wtrocki — APPROVED** [review]
Reviewed. Good to go!

**Pull request merged**